python_tests/pytest_utils.py

```diff
@@ -73,11 +88,15 @@ def make_tensor_like(a):
     )
 
 
-def make_number(dtype: torch.dtype):
+def make_number(
+    dtype: torch.dtype, low: Optional[float] = None, high: Optional[float] = None
```

Comment:
where is `low` and `high` used?

Reply:
I use `low` for the `length` and `step` arguments in `iota`. `high` isn't used but added for symmetry.

Reply:
I saw the part where low is used in test.

But argument `low` isn't passed to `make_tensor` below. Is this a bug?

Reply:
You're right. I'm blind.